lib/ddtrace/contrib/graphql/patcher.rb

```diff
@@ -0,0 +1,58 @@
+require 'ddtrace/ext/app_types'
+require 'ddtrace/ext/http'
+
+module Datadog
+  module Contrib
+    module GraphQL
+      # Provides instrumentation for `graphql` through the GraphQL tracing framework
+      module Patcher
+        include Base
+        register_as :graphql
+        option :tracer, default: Datadog.tracer
+        option :service_name, default: 'ruby-graphql' do |value|
```

Comment:
you should add here: `depends_on: [:tracer]`. that way we can ensure `tracer` option will be evaluated first